### ISSUE TITLE
Dont show feeds in Popup

### ISSUE THREAD

**Issue by Speed1** [olsh/Feedly-Notifier-Firefox#110]
Hi,
I have a problem with Feedly Notifier on Firefox. Sometimes when I get new feeds the "Popup" remains white. The only way to fix this is to restart Firefox or to change in the settings the "Popup font size in percent" settings.
![feedlynotifier](https://cloud.githubusercontent.com/assets/11215021/6396187/569c421c-bddd-11e4-8a7e-f673ce0a4a16.jpg)
I already tried to create a new Firefox Profile but without success. Maybe someone has a suggestion?
Best regards

**Comment by olsh** [comment]
Hi,
What version of Firefox do you use?

**Comment by Speed1** [comment]
Hi, it was 35.0.1 and now I'm on 36 but I have this bug already a while. Also in older versions... Maybe its a incompatibility with other Addons?

**Comment by olsh** [comment]
Yes, I think it can be so.
Please, try to experiment with disabling/enabling your other addons to detect where problem is.

**Comment by olsh** [comment]
Also try to disable [hardware acceleration](https://support.mozilla.org/en-US/kb/troubleshoot-extensions-themes-to-fix-problems#w_turn-off-hardware-acceleration).

**Comment by Speed1** [comment]
I will try and let you know. Thx!

**Comment by olsh** [comment]
Does the problem still exist?

**Comment by Speed1** [comment]
Yes, it still exist, also with Hardware Acceleration disabled.
This are my active Addons:
Dictionary Switcher
Feedly Notifier
GKeep Panel
InstantFox
Pushbullet
Search by Image (Google)
Tab Mix Plus
uBlock

**Comment by olsh** [comment]
I installed all these addons, but I've not managed to reproduce the problem.

**Comment by Speed1** [comment]
Yes its strange because it happens only sometimes (once a day)... bo, maybe some bug with my notebook.

**Comment by dorugeber** [comment]
FWIW this hasn't happened to me for _months_ now.  Can probably be closed.

**Comment by olsh** [comment]
@dorugeber thanks! 👍
